docs/niswitch/functions.rst

```diff
@@ -782,7 +782,7 @@ niswitch.Session methods
         
 
 
-    :type maximum_time_ms: int
+    :type maximum_time_ms: float in seconds or datetime.timedelta
 
 .. py:method:: wait_for_scan_complete(maximum_time_ms=datetime.timedelta(milliseconds=5000))
```

Comment:
Observation: Based on our defaults, we are "suggesting" clients use timedelta.

Reply:
Yes. Should we somehow document this somewhere? If so, where?

Reply:
I think this is enough. It's just an observation. If we are nudging one way or another, I think we should nudge consistently. This means to keep an eye open for things such as using timeouts in examples in the form of timedeltas.